modules/mint-mode-with-removal-of-admin-credential.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/manually-creating-iam.adoc
+
+[id="mint-mode-with-removal-or-rotation-of-admin-credential_{context}"]
+= Mint mode with removal or rotation of the admin credential
+
+In this mode, a user installs {product-title} with an `admin` credential just
+like the normal mint mode. However, this mode removes the `admin` credential
+secret from the cluster post-installation. The Cloud Credential Operator makes
```

Comment:
Could we clarify that this is an action that must be taken by a human administrator.